src/WindowsUtils/include/WindowsUtils/Injection.h

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
```

Comment:
Is this file duplicated now? Or is it again weird GitHub behaviour?

Reply:
Not sure what happened there, removed the file.